core/src/main/java/org/bitcoinj/core/VersionMessage.java

```diff
@@ -113,6 +113,15 @@ public VersionMessage(NetworkParameters params, int newBestHeight) {
         relayTxesBeforeFilter = true;
     }
 
+    /**
+     * Get the service bitfield that represents the node services being provided.
+     *
+     * @return service bitfield
+     */
+    public Services getServices() {
```

Comment:
Perhaps this should be `.services()`?  If not now, then perhaps after `localServices` can be made final.

Reply:
At the moment, it's mutable. So maybe we continue accessing the field for now and not add the getter until we tackle the immutability?